x/pool-incentives/keeper/keeper.go

```diff
@@ -63,26 +63,21 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 	return ctx.Logger().With("module", "x/"+types.ModuleName)
 }
 
-// CreatePoolGauges checks if the poolId is Concentrated Liquidity PoolType, if it is create one gauge,
-// otherwise create multiple gauges based on lockableDurations.
-func (k Keeper) CreatePoolGauges(ctx sdk.Context, poolId uint64) error {
```

Comment:
Note: what was done here is splitting logic into two separate functions.

One that is called by the `AfterCLPoolCreated`. One that is called by the `AfterCFMMPoolCreated`